### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by marxeille**

### CODE REVIEW

File: packages/mobile/src/screens/stake/redelegate/index.tsx [hunk 1]
```diff
@@ -423,20 +425,76 @@ export const RedelegateScreen: FunctionComponent = observer(() => {
...
+              keyboardType={'numeric'}
+              labelStyle={{
+                fontSize: 16,
+                fontWeight: '700',
+                lineHeight: 22,
+                color: colors['gray-900'],
+                marginBottom: spacing['8']
+              }}
+              onChangeText={text => {
+                const fee = new Dec(Number(text.replace(/,/g, '.')) * 10 ** 6);
```
Comment (tubackkhoa) [inline packages/mobile/src/screens/stake/redelegate/index.tsx]: `new Dec(Number(text.replace(/,/g, '.')).mul(Dec.getPrecisionDec(6))`

File: packages/mobile/src/screens/stake/undelegate/index.tsx [hunk 2]
```diff
@@ -156,12 +161,68 @@ export const UndelegateScreen: FunctionComponent = observer(() => {
...
+          keyboardType={'numeric'}
+          labelStyle={{
+            fontSize: 16,
+            fontWeight: '700',
+            lineHeight: 22,
+            color: colors['gray-900'],
+            marginBottom: spacing['8']
+          }}
+          onChangeText={text => {
+            const fee = new Dec(Number(text.replace(/,/g, '.')) * 10 ** 6);
```
Comment (tubackkhoa) [inline packages/mobile/src/screens/stake/undelegate/index.tsx]: `new Dec(Number(text.replace(/,/g, '.')).mul(Dec.getPrecisionDec(6))`

**Review by im6h — DISMISSED** [review]

File: packages/mobile/src/screens/stake/redelegate/index.tsx (hunk 1 above)
Comment (marxeille) [reply packages/mobile/src/screens/stake/redelegate/index.tsx]: Number does not have mul(Dec.getPrecisionDec(6), so i replace it with Math.pow

File: packages/mobile/src/screens/stake/undelegate/index.tsx (hunk 2 above)
Comment (marxeille) [reply packages/mobile/src/screens/stake/undelegate/index.tsx]: Number does not have mul(Dec.getPrecisionDec(6), so i replace it with Math.pow

**Review by tubackkhoa — APPROVED** [review]

**Review by im6h — APPROVED** [review]

**Review by im6h — APPROVED** [review]

**Review by im6h — APPROVED** [review]

**Review by im6h — APPROVED** [review]

**Review by haunv3 — APPROVED** [review]
lgtm

**Pull request merged**